twentytwentyone-blocks/assets/css/blocks.css

```diff
@@ -59,11 +59,23 @@
 # Image
 --------------------------------------------------------------*/
 
-.wp-block-image img,
 .wp-block-image img {
 	height: auto;
 }
 
+.wp-block-image figcaption{
+	text-align: center;
+	color: var(--wp--preset--color--dark-gray);
+	font-size: var(--wp--preset--font-size--extra-small);
+	line-height: var(--wp--custom--line-height--body);
+	margin-top: calc(0.5 * var(--wp--custom--spacing--unit));
+	margin-bottom: var(--wp--custom--spacing--unit);
+}
+
+.wp-block-image a:focus img {
```

Comment:
I think we can leave this out until we have a resolution to https://github.com/WordPress/gutenberg/issues/27075 happens. We don't currently have any styles in the heme that would add an `outline` on focus anyway.